src/static/js/modules/jquery/cf_formValidator.js

```diff
@@ -0,0 +1,181 @@
+/* ==========================================================================
+   jquery.cf_formValidator
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+var _validate = require( 'validate' );
```

Comment:
I may have misspoken about underscore prefixing imported modules. Will need to check w/ node conventions.